ykman/cli/slot.py

```diff
@@ -170,57 +170,90 @@ def delete(ctx, slot, force):
               callback=parse_hex(16), help='16 byte secret key, in hex.')
 @click.option('--no-enter', is_flag=True, help="Don't send an Enter "
               'keystroke after outputting an OTP.')
+@click.option(
```

Comment:
Do we need the alias command? We don't have that anywhere else. I'd prefer coming up with short flags for the below flags instead.

Reply:
Yeah, good point. The main reason I added it was actually as a way to decouple the [`test_update_settings_enter_slot_2`][test1] and [`test_delete_slot_2`][test2] tests from the internal logic a bit, but I think I could as well make them use a static password slot instead.

[test1]: https://github.com/Yubico/yubikey-manager/pull/79/files/e1ec2ed89e94ae95505a9342446ab8a75f5243c3#diff-c943060aa5353f04a2721936c20d2a33L174
[test2]: https://github.com/Yubico/yubikey-manager/pull/79/files/e1ec2ed89e94ae95505a9342446ab8a75f5243c3#diff-c943060aa5353f04a2721936c20d2a33L179

Reply:
Fixed.